chartmogul.go

```diff
@@ -85,7 +85,7 @@ type IApi interface {
 	SearchCustomers(SearchCustomersParams *SearchCustomersParams) (*Customers, error)
 	MergeCustomers(MergeCustomersParams *MergeCustomersParams) error
 	DeleteCustomer(customerUUID string) error
-	DeleteCustomerInvoices(dataSourceUUID, customerUUID string) error
+	DeleteCustomerInvoices(dataSourceUUID, customerUUID string, customerExternalID ...string) error
```

Comment:
I know it might seem like splitting hair, but this has some disadvatages:
1) you can pass multiple `external_ids`, but the code ignores the rest.
2) the library uses structs with fields for optional parameters - eg. `api.SearchCustomers(&cm.SearchCustomersParams{})`, it would be better to keep one pattern.

Your way would help being backwards compatible, however we can achieve it with creating the new function signature as `DeleteCustomerInvoicesV2`.